doc/source/user_guide/numpy_images.rst

```diff
@@ -13,6 +13,17 @@ manipulating arrays::
     >>> type(camera)
     <type 'numpy.ndarray'>
 
+.. note::
+    Complex array-like types, such as
```

Comment:
```suggestion
    Rich array-like data types, such as
```
Hesitating between "Rich" and "Labeled" (`pandas` and `xarray` mention 'labeled' first and foremost).

Reply:
"Labeled" sounds great, thanks for the suggestion!